patches/playroom+0.28.0.patch

```diff
@@ -36,6 +36,15 @@ index 56defa7..1e7cf3b 100644
        },
      },
      module: {
+@@ -163,7 +166,7 @@ module.exports = async (playroomConfig, options) => {
+   };
+ 
+   const theirConfig = playroomConfig.webpackConfig
+-    ? await playroomConfig.webpackConfig()
++    ? await playroomConfig.webpackConfig(options)
```

Comment:
playroom doesn't yet do this for some reason. Small change but better for DX imo (instead of referencing a separate env var which may have side-effects on other parts of our build). 

Aiming to contrib this back to playroom next week